src/app/services/todo/todo.service.ts

```diff
@@ -0,0 +1,4 @@
+export interface TodoService {
+  addTodo: (name: string) => void;
```

Comment:
All service methods should return Promise. 

This ensures that in future it can perform any network I/O if needed.

Also `addTodo ` should return actual TODO item object


addTodo: (name: string) => Promise<TODOItem>